Misc/NEWS.d/next/Library/2024-06-12-04-44-13.gh-issue-120381.GwTGOV.rst

```diff
@@ -0,0 +1,2 @@
+Correct ``inspect.ismethoddescriptor`` to check also for the lack of
```

Comment:
```suggestion
Correct :meth:`inspect.ismethoddescriptor` to check also for the lack of
```

Reply:
OK, fixed both references (actually, the first one is a `:func:` one, and the second one is a `:meth:` one).